internal/formats/spdx22json/to_format_model.go

```diff
@@ -37,14 +38,34 @@ func toFormatModel(s sbom.SBOM) *model.Document {
 			},
 			LicenseListVersion: spdxlicense.Version,
 		},
-		DataLicense:       "CC0-1.0",
-		DocumentNamespace: namespace,
-		Packages:          toPackages(s.Artifacts.PackageCatalog, s.Relationships),
-		Files:             toFiles(s),
-		Relationships:     toRelationships(s.Relationships),
+		DataLicense:          "CC0-1.0",
+		ExternalDocumentRefs: toExternalDocumentRefs(s.Relationships),
+		DocumentNamespace:    namespace,
+		Packages:             toPackages(s.Artifacts.PackageCatalog, s.Relationships),
+		Files:                toFiles(s),
+		Relationships:        toRelationships(s.Relationships),
 	}
 }
 
+func toExternalDocumentRefs(relationships []artifact.Relationship) []model.ExternalDocumentRef {
```

Comment:
nit: if `relationships` is nil/empty then we will return a nil `[]model.ExternalDocumentRef`

Do we want this value to be nil when constructing `model.Document or is it more friendly for the output for this to be `[]model.ExternalDOcumentRef{}` with `len == 0`?

Reply:
Yeah makes sense. Fixed for this and also for `toRelationships`